tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -32,11 +32,21 @@ std::uint64_t GetNewModelId()
 }
 } // anonymous namespace
 
+ROOT::Experimental::RFieldZero &ROOT::Experimental::Internal::GetFieldZero(ROOT::Experimental::RNTupleModel &model)
+{
+   return *model.fFieldZero;
+}
+
+ROOT::Experimental::Internal::RProjectedFields &
+ROOT::Experimental::Internal::GetProjectedFields(ROOT::Experimental::RNTupleModel &model)
+{
```

Comment:
We may also want to also add a new state akin to 'ReadyToUse' after all ids, etc.. have been put in place and protect from inadvertent future mistake and add:
```suggestion
{
     if (model.ReadyToUse())
         throw RException(R__FAIL("invalid attempt to get projected zero field of ReadyToUse model"));
```

Reply:
I think this is not for this PR.

Reply:
Fair enough but I think it is necessary to put it as soon as possible.